create_references_for_ntjoin.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Split a fasta file into multiple files, one per sequence and create a reference config file for ntjoin
+# Usage: split_fasta.py <fasta file> <output directory> <config file>
+
+import sys
+
+input = sys.argv[1]
+output = sys.argv[2]
+config = sys.argv[3]
+ref_list = []
+with open(input, 'r') as f:
+    lines = f.readlines()
+    for i in range(len(lines)):
+        if lines[i].startswith('>'):
+            header = lines[i].strip()
+            filename = header.split(' ')[0].replace('>', '') + '.fasta'
+            sequence = lines[i+1].strip()
+            with open(output + '/' + filename, 'w') as g:
+                g.write(header + '\n')
+                g.write(sequence + '\n')
+            ref_list.append(f'{output}/{filename},2')
```

Comment:
I'd make this a variable at the top of the file - it'll be easier in the future to remember what this number is for our future selves, and a tad more straightforward for someone to change if needed

Reply:
Did you mean creating a `reference_weight` variable? Do you think there will be situations where references need to be assigned different weight values? Or do you think it is not necessary?

Reply:
Yes, that's what I mean! This is a 'magic number' as-is (like when a number seems to come out of nowhere), so giving it a variable name instead makes the code more readable